feat: add more variants of varsities

Ksargen opened the pull request:
New variants include one for each department: Supply, RnD, Engineering, Medical and Security
Available for every bodytype

rufuszero commented on code/modules/clothing/suits/miscellaneous.dm:
```diff
@@ -382,6 +382,41 @@
+
+/obj/item/clothing/suit/storage/toggle/varsity/cargo
+	name = "cargo varsity"
+	desc = "An old, worn but loyal and reliable varsity jacket in faded white and yellow. Comes with extra pockets for convenience!"
```
>Comes with extra pockets

Does it really? The description could be confusing in terms of the actual storage mechanics.

Ksargen replied:
It does visually, but not mechanically. I totally forgot about actual pocket mechanic and can see why this could be confusing. I guess I will either delete this part or change to "Comes with extra pockets for visual display only"

rufuszero commented on code/modules/clothing/suits/miscellaneous.dm:
```diff
@@ -382,6 +382,41 @@
+
+/obj/item/clothing/suit/storage/toggle/varsity/eng
+	name = "engineering varsity"
+	desc = "A yellow and orange varsity jacket with a lightning bolt and a gear symbol—the emblem of the fraternity (or sorority)—on the back. Funny enough, there are no gears on a space station."
```
> Funny enough

Either use "funnily enough" or "ironically".

I would also recommend using simpler English sentences for the description instead of the "...—...—..." constructions. Especially since our primary audience are not native English speakers and we try to simplify the sentences where possible.

Ksargen replied:
Well I have pretty small actual english writing/speaking experience so I will trust you here. I will use 'ironically'
Also not really relevant for this particular conversation, but descriptions are not AI generated, just machine translated. Copy-pasted without putting much attention, so my bad

rufuszero commented on code/modules/clothing/suits/miscellaneous.dm:
```diff
@@ -382,6 +382,41 @@
 
+/obj/item/clothing/suit/storage/toggle/varsity/med
+	name = "medical varsity"
+	desc = "A varsity jacket in blue, red, and white, featuring an image of the Caduceus on the back, symbolizing not only the art of healing but also your fraternity (or sorority)"
```
You can drop the redundant parentheses, leaving just "...your fraternity or sorority."

There should be a comma before "but".

Feel free to use this simplified version which keeps the wording pretty much the same.

> A varsity jacket in blue, red, and white, featuring an image of the Caduceus on the back, symbolizing both the art of healing and your fraternity or sorority.

Ksargen replied:
Got it with parentheses. 
Punctuation in English is somewhat mystery for me, but this looks like my blunder
I will use your version

rufuszero reviewed (changes requested):
It'd be ideal to have at least some basic screenshots attached to the PR. Best practice, of course, is to provide in-game examples in various environments and lighting, but even basic screenshots from your .dmi editor of choice will be helpful.

Even though I've seen them in our Discord chat, future contributors won't have any way to tell what the added sprites look like without downloading and opening .dmi files individually.

Also added a few description nitpicks.
The new descriptions feel AI generated, which isn't critical if they are, but if my guess is right, it could as well be obvious to others. Ping me on Discord if you're interested in discussing how we might improve them. To me, these descriptions feel quite disconnected from the general text style of the game, and I'm happy to help enhance the quality of the game with a more tailored approach.

Ksargen commented:
Forgot to add description to commit
Rearranged structure of .dmi files: now all varsities are grouped together
Changed descriptions for Cargo, Security, Engineering and Science varsities
Changed sprites in closed state for Fat bodybuild
Made new design on the back for Science varsities
Removed holes in Cargo varsities
Made cog on the back of Engineering varsities a bit more contrast

![photo_2025-01-03_02-37-00](https://github.com/user-attachments/assets/e31ff465-c682-4aed-803c-803fc930355a)
![photo_2025-01-03_02-36-58](https://github.com/user-attachments/assets/131131dc-2691-4a39-8b1a-fc0daae71a1f)
![photo_2025-01-03_02-36-57](https://github.com/user-attachments/assets/b9f6dea9-fc8e-4b29-ba13-d22ee71a7505)
![photo_2025-01-03_02-36-55](https://github.com/user-attachments/assets/30193946-190b-49cf-b864-6beeb965a8db)
![photo_2025-01-03_02-36-53](https://github.com/user-attachments/assets/1176c9d8-2218-4b53-9b3e-ddcaffe7351a)
![photo_2025-01-03_02-37-06](https://github.com/user-attachments/assets/61e59cb9-eed4-4724-b7c2-0cd134319e36)
![photo_2025-01-03_02-37-05](https://github.com/user-attachments/assets/a008cafe-e8d6-4b87-9c31-038c58b561b5)
![photo_2025-01-03_02-37-03](https://github.com/user-attachments/assets/ad76a8d9-39ed-4cee-ba8f-53946fadeaad)

rufuszero commented:
Suggested changes:
### Cargo:
> An old, worn varsity jacket in faded yellow and white, patched up a couple of times. It's loyal and reliable, features multiple pockets, though only two are deep enough to actually store anything.

Some rearrangements for the sake of readability and consistency.

### Security:
> A varsity in red, white, and black with a scorpion design on the back and a patch in the shape of a sheriff's badge on the front. This jacket was given only to the top drivers of the fraternity or sorority.

Missed article "a" before "sheriff's"

### Science:
A flask design on the back maybe? Similar to this one:
![image](https://github.com/user-attachments/assets/9fd90af8-52ae-47f5-91d0-01b485b621cc)

A zero is also fine by me, gives it some quirkiness that players might roleplay around of.

### Engineering:
> A yellow and orange varsity with a zipper, a cold and hot counter patch on the shoulder, and a gear design on the back, symbolizing your allegiance to the fraternity or sorority. Ironically, there are no gears on a space station.

Grammar fixes.

Ksargen commented:
I tried to come up with something for a science varsity, but I couldn't come up with anything that wouldn't stand out and look like a flask. The work area was too small for me.

rufuszero commented:
@Ksargen it's better to rebase feature branches over master than merge them. Keeps the PR history clean from the useless "merge master into varsities" commits that will appear every time you update the branch. I can teach you how to properly rebase on your own if you're interested.

rufuszero commented:
Tested these in the game and they all look good to go. Great colorful and stylish addition to the crew's wardrobe.
Thanks for your patience on this PR's reviews!

Ksargen commented:
Yeah, some insight into rebasing would be useful
